R/wrappers.R

```diff
@@ -497,16 +517,25 @@ prop_test <- function(x, formula,
 
   # two sample
   if (has_explanatory(x)) {
-
-    order <- check_order(x, order, in_calculate = FALSE, stat = NULL)
-
     # make a summary table to supply to prop.test
     sum_table <- x %>%
-      select(response_name(x), explanatory_name(x)) %>%
-      table()
-
-    # reorder according to the order and success arguments
-    sum_table <- sum_table[lvls, order]
+       select(explanatory_name(x), response_name(x)) %>%
```

Comment:
It's not obvious to me how we could formally test which selection order is "right," though I'm quite confident it's correct now. :)